publish.sh

```diff
@@ -45,11 +45,11 @@ then
   echo "creating repo tag for $RELEASE_TYPE release"
   ./gradlew -Prelease.scope=$RELEASE_SCOPE clean $RELEASE_TYPE
   echo "publishing $REPOSITORY artifacts for $RELEASE_SCOPE $RELEASE_TYPE release"
-  ./gradlew -Prelease.useLastTag=true -PallScalaVersions $PUBLISH_TASK
+  ./gradlew -Prelease.useLastTag=true $PUBLISH_TASK
```

Comment:
i guess we could safely leave this as is, in case we ever start cross compiling again. your call.